packages/marked-mathjax/src/index.ts

```diff
@@ -15,6 +15,25 @@ const startMath = /(\$|\\\(|\\\[)/;
 export function addMathjaxExtension(marked: Marked, MathJax: any) {
   const mathjaxInput = MathJax.startup.getInputJax() ?? [];
   marked.use({
+    hooks: {
+      // This hook ensures that, if a math token is to be found inside an
+      // em/strong token, no `*` or `_` characters are interpreted as em/strong
+      // termination. This is done by masking the content that the em/strong
+      // interpreter use to find the termination characters.
+      emStrongMask(src) {
+        // Fast-path: no potential math delimiters present
+        if (!startMath.test(src)) return src;
+        for (const inputJax of mathjaxInput) {
+          for (const foundMath of inputJax.findMath([src])) {
+            src =
+              src.slice(0, foundMath.start.n) +
+              'a'.repeat(foundMath.end.n - foundMath.start.n) +
```

Comment:
What does "mask" mean here? I see here and in your PR to Marked that you're using a long string of `a` characters, but I don't see any description of why, or what Marked will do with that.